src/components/TodoList.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { ITodo } from '../types/ITodo';
+import './TodoList.scss';
+import { Todo } from './Todo';
+
+type Props = {
+  todos: ITodo[],
+};
+
+export const TodoList: React.FC<Props> = ({ todos = [] }) => (
```

Comment:
In your props type is required field, so there is no need to set default value
```suggestion
export const TodoList: React.FC<Props> = ({ todos }) => (
```

Reply:
fixed